deploy/helm/sumologic/values.yaml

```diff
@@ -3764,6 +3764,18 @@ metadata:
       ## The value of this key should be a dictionary, that will replace the normal configuration.
       ## This is an advanced feature, use with caution, and review the generated configuration first.
       override: {}
+      ## Includes additional processors into pipelines.
+      ## It can be used for filtering metrics, renaming, changing metadata and so on.
+      ## This is list of objects, for example:
+      ## extraProcessors:
+      ##   - filterprocessor:
+      ##       exclude:
+      ##         match_type: strict
+      ##         metric_names:
+      ##           - hello_world
+      ##           - hello/world
+      extraProcessors: []
```

Comment:
I wonder if this shouldn't live under `sumologic.metrics.metadata` instead. :thinking: It involves OT config internals, but I imagine it's also going to be one of the main ways of customizing the configuration for users.

Reply:
I was considering `metadata.metrics` as Otel specific configuration and extraProcessors is definitely Otel specific

Reply:
I guess it's ok until we expose simpler higher-level ways of doing this, like namespace regexes and so forth.